app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,29 @@
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <a class="navbar-brand" href="#">Brand</a>
+    </div>
+
+    <div class="collapse navbar-collapse">
+      <ul class="nav navbar-nav navbar-right">
+        <% if user_signed_in? %>
+          <li><a href="#" class="navbar-link no-pointer">Welcome, <%= current_user.name %></a></li>
+          <li><%= link_to 'Branch', branches_path %></li>
+          <li><%= link_to 'Management Company', companies_path %></li>
+          <li><%= link_to 'Property Manager', managers_path %></li>
+          <li><%= link_to 'Property', properties_path %></li>
+          <li><%= link_to 'Sign Out', destroy_user_session_path, :method => :delete %></li>
```

Comment:
The newer syntax allows for these hash rockets `=>` to be swapped for a cleaner `:`.  For example:

```ruby
method: :delete
```